packages/block-editor/CHANGELOG.md

```diff
@@ -18,7 +22,7 @@
 
 ## 12.18.0 (2024-01-24)
 
-- Deprecated `__experimentalRecursionProvider` and `__experimentalUseHasRecursion` in favor of their new stable counterparts `RecursionProvider` and `useHasRecursion`.
+-   Deprecated `__experimentalRecursionProvider` and `__experimentalUseHasRecursion` in favor of their new stable counterparts `RecursionProvider` and `useHasRecursion`.
```

Comment:
Are the white space changes in this file intentional?

Reply:
They were not formatted correctly and my IDE applied the formatting automatically. Seems reasonable to leave it IMO.